content/en/_index.md

```diff
@@ -36,99 +36,99 @@ section2:
+      content: Provide a web console for creating and managing Kubernetes resources, with powerful observability
 
     - name: Multi-tenant Management 
       icon: /images/home/multi-tenant-management.svg
-      content: Provide unified authentication with fine-grained roles and three-tier authorization system, supports AD/LDAP authentication
+      content: Provide unified authentication with fine-grained roles and three-tier authorization system, and support AD/LDAP authentication
   
   features:
     - name: Application Store 
       icon: /images/home/store.svg
-      content: Provide application store for Helm-based applications, and offers application lifecycle management
+      content: Provide an application store for Helm-based applications, and offer application lifecycle management
       color: grape
 
     - name: Service Mesh (Istio-based)
       icon: /images/home/service.svg
-      content: Provide fine-grained traffic management, observability and tracing, and offers visualization for traffic topology 
+      content: Provide fine-grained traffic management, observability and tracing, and offer visualization for traffic topology 
       color: red
 
     - name: Rich Observability
       icon: /images/home/rich.svg
-      content: Multi-dimensional monitoring metrics, multi-tenant log query and collection, support alerting and notification
+      content: Multi-dimensional monitoring metrics, multi-tenant log query and collection, and alerting and notification support
       color: green
 
     - name: DevOps System
       icon: /images/home/dev-ops.svg
-      content: Out-of-box CI/CD based on Jenkins, and offers automated workflow tools including S2I & B2I
+      content: Out-of-box CI/CD based on Jenkins, and automated workflow tools including S2I & B2I
       color: orange
 
     - name: Multiple Storage Solutions
       icon: /images/home/multiple.svg
-      content: Support GlusterFS, CephRBD, NFS, LocalPV solutions, provide CSI plugins to consume storage from multiple cloud providers
+      content: Support GlusterFS, CephRBD, NFS, LocalPV solutions, and provide CSI plugins to consume storage from multiple cloud providers
       color: grape
 
     - name: Multiple Network Solutions 
       icon: /images/home/network.svg
-      content: Support Calico and Flannel, provides load balancer plug-in Porter for Kubernetes installed on physical machines
+      content: Support Calico and Flannel, and provide load balancer plug-in Porter for Kubernetes installed on physical machines
       color: green
 
     - name: Multi-cluster management 
       icon: /images/home/management.svg
-      content: Distribute applications across multiple clusters and cloud providers, and provides the disaster recovery and cross-cluster discovery.
+      content: Distribute applications across multiple clusters and cloud providers, and provide disaster recovery plans and cross-cluster discovery
```

Comment:
I suggest to replace `plans` with `solutions`

Reply:
done